app/assets/javascripts/components/overview/course_stats.jsx

```diff
@@ -3,14 +3,41 @@ import PropTypes from 'prop-types';
 import CourseUtils from '../../utils/course_utils.js';
 
 const CourseStats = ({ course }) => {
+  const isWikidata = course.home_wiki.project === 'wikidata';
+  const isWikipedia = course.home_wiki.project === 'wikipedia';
+
   const valueClass = (stat) => {
     if (!course.newStats) { return 'stat-display__value'; }
     return course.newStats[stat] ? 'stat-display__value stat-change' : 'stat-display__value';
   };
 
-  let viewData;
+  let editedLabel;
+  let createdLabel;
+  if (isWikidata) {
+    createdLabel = I18n.t('metrics.articles_created_wikidata');
+    editedLabel = I18n.t('metrics.articles_edited_wikidata');
+  } else if (isWikipedia) {
+    createdLabel = I18n.t('metrics.articles_created');
+    editedLabel = I18n.t('metrics.articles_edited');
+  } else {
+    createdLabel = I18n.t('metrics.articles_created_generic');
+    editedLabel = I18n.t('metrics.articles_edited_generic');
+  }
+
+  let pageviews;
   let infoImg;
   let trainedTooltip;
+  let uploadCount;
+
+  let articlesCreated;
+  if (course.created_count !== '0') {
```

Comment:
Hm. Would this get the same idea across? Just a suggestion.
```js
const count = Number(course.created_count)
if (count > 0) { // or, (count !== 0)
// ...
}
```

Reply:
That works mostly, except sometimes created_count could be something like '1.2k'.

Reply:
Ah, that makes sense. Ok, well, enh. Thanks for clarifying.